src/sentry/feedback/blueprints/api.md

```diff
@@ -0,0 +1,315 @@
+        "feedback_id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
+        "locale": {
+          "lang": "en",
+          "timezone": "UTC+1"
+        },
+        "message": "I really like this feedback feature!",
+        "os": {
+          "name": "iOS",
+          "version": "16.2"
+        },
+        "platform": "javascript",
+        "project_id": 11276,
+        "release": "version@1.3",
+        "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
+        "sdk": {
+          "name": "sentry.javascript.react",
+          "version": "6.18.1"
+        },
+        "status": "unresolved",
+        "tags": {
+          "hello": "is",
+          "it": ["me", "you're", "looking", "for"]
+        },
+        "timestamp": "2023-08-31T14:10:34.954048",
+        "url": "https://docs.sentry.io/platforms/javascript/",
+        "user": {
+          "display_name": "John Doe",
+          "email": "john.doe@example.com",
+          "id": "30246326",
+          "ip": "213.164.1.114",
+          "username": "John Doe"
+        }
+      }
+    }
+    ```
+
+### Fetch Feedback [GET]
+
+Retrieve a single feedback item.
+
+- Response 200
+
+  [UserFeedback]
+
+### Update Feedback [PATCH]
+
+Partially update a feedback item.
+
+- Request
+
+  ```json
+  {
+    "data": {
+      "status": "resolved"
+    }
+  }
+  ```
+
+- Response 202
+
+  [UserFeedback]
+
+### Delete Feedback [DELETE]
+
+Delete a feedback item.
+
+- Response 204
+
+## Feedback Ingest [/feedback/]
+
+### Create Feedback [POST]
+
+Create a new feedback item. This method is a subset of the Event protocol with
+an additional "feedback" object added. Within the feedback object are feedback
+related event metadata.
+
+Every field not marked as "optional" is considered a required field and must be
+present in the request body.
+
+See https://develop.sentry.dev/sdk/event-payloads/types/ for more information
+
+| Column                     | Type             | Description                           |
+| -------------------------- | ---------------- | ------------------------------------- |
+| dist                       | optional[string] | -                                     |
+| environment                | string           | -                                     |
+| event_id                   | optional[string] | Omitted IDs are internally generated. |
+| feedback                   | object           | -                                     |
+| feedback.contact_email     | string           | -                                     |
+| feedback.message           | string           | -                                     |
+| feedback.replay_id         | optional[string] | -                                     |
+| feedback.url               | string           | -                                     |
+| platform                   | string           | -                                     |
+| release                    | string           | -                                     |
```

Comment:
Can we make environment and release optional? they're not always set. cc @cmanallen @michellewzhang 

```suggestion
| environment                | optional[string]           | -                                     |
| event_id                   | optional[string] | Omitted IDs are internally generated. |
| feedback                   | object           | -                                     |
| feedback.contact_email     | string           | -                                     |
| feedback.message           | string           | -                                     |
| feedback.replay_id         | optional[string] | -                                     |
| feedback.url               | string           | -                                     |
| platform                   | string           | -                                     |
| release                    | optional[string]           | -                                     |
```

Reply:
yup, I'll update the endpoints right now